collector/mongos/database_status.go

```diff
@@ -109,15 +109,15 @@ func GetDatabaseStatList(client *mongo.Client) *DatabaseStatList {
 	dbStatList := &DatabaseStatList{}
 	dbNames, err := client.ListDatabaseNames(context.TODO(), bson.M{})
 	if err != nil {
-		log.Error("Failed to get database names")
+		log.Error(err)
 		return nil
 	}
 	for _, db := range dbNames {
 		dbStatus := DatabaseStatus{}
 		r := client.Database(db).RunCommand(context.TODO(), bson.D{{"dbStats", 1}, {"scale", 1}})
 		err := r.Decode(&dbStatus)
 		if err != nil {
-			log.Error("Failed to get database status.")
+			log.Error(err)
```

Comment:
That will log something like "context deadline exceeded" without any context (pun intended). I would say we should use both message and error:
```go
log.Errorf("Failed to get database status: %s.", err)
```